backpack/core/derivatives/conv_transposend.py

```diff
@@ -52,6 +52,9 @@ def _bias_jac_mat_prod(self, module, g_inp, g_out, mat):
         return jac_mat.expand(*expand_shape)
 
     def _weight_jac_mat_prod(self, module, g_inp, g_out, mat):
+        if module.groups != 1:
+            raise NotImplementedError
```

Comment:
Can you add a message to the exeception? Something like "groups ≠ 1 is not supported"